src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
Let me merge this one. Then, if you finish the **container** enhancement, you can submit another PR. Does that sound reasonable?

> BTW, thanks for all your great comments, issues, research, and PR(s).